pkg/sentry/platform/kvm/kvm.go

```diff
@@ -117,15 +117,7 @@ func (*KVM) MaxUserAddress() usermem.Addr {
 func (k *KVM) NewAddressSpace(_ interface{}) (platform.AddressSpace, <-chan struct{}, error) {
 	// Allocate page tables and install system mappings.
 	pageTables := pagetables.New(newAllocator())
-	applyPhysicalRegions(func(pr physicalRegion) bool {
-		// Map the kernel in the upper half.
-		pageTables.Map(
-			usermem.Addr(ring0.KernelStartAddress|pr.virtual),
-			pr.length,
-			pagetables.MapOpts{AccessType: usermem.AnyAccess},
-			pr.physical)
-		return true // Keep iterating.
-	})
+	pageTables.MirrorKernelPageTable()
```

Comment:
> Per elsewhere, I think this would look something like:
> 
> pageTables.Mirror(m.kernel.PageTables, kernelStart, ....)

Thanks for advise on the patch, add new method Mirror, and take range as parameter, also using the walk API, I will try to update the patch later.